iroh-car/src/util.rs

```diff
@@ -0,0 +1,73 @@
+use cid::Cid;
+use integer_encoding::VarIntAsyncReader;
+use tokio::io::{AsyncRead, AsyncReadExt};
+
+use super::error::Error;
+pub(crate) async fn ld_read<R>(mut reader: R, buf: &mut Vec<u8>) -> Result<bool, Error>
```

Comment:
Is this called `ld_read` because it read `length` first and then `data`? If so, is that a common abbreviation that I'm just unfamiliar with? Cause it was a bit confusing!

Reply:
ld stands for lentgth delimited